tobac/segmentation.py

```diff
@@ -286,10 +310,33 @@ def segmentation(
 
 
 def watershedding_3D(track, field_in, **kwargs):
+    warnings.simplefilter(
+        "always", DeprecationWarning
+    )  # turn off filter; this is an easy switch for users.
+    warnings.warn(
+        "Warning: using deprecated function `watershedding_3D`. Switch to `segmentation` instead."
+        " The arguments and output of `segmentation` is the same, so it is an easy switch."
+        " This function will be removed in v1.5.0.",
+        category=DeprecationWarning,
+        stacklevel=2,
+    )
+    warnings.simplefilter("default", DeprecationWarning)  # reset filter
+
     kwargs.pop("method", None)
     return segmentation_3D(track, field_in, method="watershed", **kwargs)
 
 
 def watershedding_2D(track, field_in, **kwargs):
+    warnings.simplefilter(
+        "always", DeprecationWarning
+    )  # turn off filter; this is an easy switch for users.
+    warnings.warn(
+        "Warning: using deprecated function `watershedding_2D`. Switch to `segmentation` instead."
+        " The arguments and output of `segmentation` is the same, so it is an easy switch."
```

Comment:
With the tiny difference that for `watershedding_2D` and `watershedding_3D` `dxy` is a key word argument. Do we need to mention that or not?

Reply:
I think I have resolved this with the new commits- `watershedding_2D` and `watershedding_3D` now call `segmentation` directly, and if `dxy` isn't included, a `ValueError` is raised, as it is needed down the line.